### DISCUSSION TITLE
Barbar buffers order support

### DISCUSSION THREAD

**Discussion by Martin1887** [olimorris/persisted.nvim#118] [Ideas]
Thanks for this great plugin!

The only inconvenience I have found is the order of barbar buffers is not saved. It includes a snippet for persistence.nvim, but I'm not sure how something similar would work for persistence.nvim. I have tried to simply add the globals to the session options but it does not work.

The barbar snippet:
https://github.com/romgrk/barbar.nvim?tab=readme-ov-file#persistencenvim

Thanks!

**Comment by olimorris** [comment]
If It can't be saved in a session as per `:h sessionoptions` then the plugin can't support it.

Ref: https://github.com/olimorris/persisted.nvim?tab=readme-ov-file#what-is-saved-in-the-session
